doc/Language/ipc.pod6

```diff
@@ -102,4 +102,4 @@ Use the C<write> method to pass data into the program.
 
 =end pod
 
-# vim: expandtab softtabstop=4 shiftwidth=4 ft=perl6
\ No newline at end of file
+# vim: expandtab softtabstop=4 shiftwidth=4 ft=perl6
```

Comment:
Don't see what's the change here, but as long as it's being changed, maybe change to ft=raku (if that works with vim)

Reply:
This change was adding a newline at the end of the file.